index.d.ts

```diff
@@ -1781,5 +1776,53 @@ export namespace stream {
 			addWorkbookRels(): Promise<void>;
 			addWorkbook(): Promise<void>;
 		}
+
+
+		interface WorkbookStreamReaderOptions {
+			/**
+			 * @default 'emit'
+			 */
+			entries: 'emit' | void;
```

Comment:
TS has two ways to define optional parameters like here:
```typescript
			first: 'emit' | 'cache' | void;
//or
			second?: 'emit' |  'cache';
```
To be honest, I will prefer the second way (with a question mark).

The most perfect way is use separate types for each property:
https://github.com/exceljs/exceljs/pull/1199/files#diff-b52768974e6bc0faccb7d4b75b162c99R995